lib/index.js

```diff
@@ -22,5 +24,13 @@ module.exports = {
   DiscNumber: require("./discNumber"),
   LsrOrder: require("./lsrOrder"),
   Lidbs: require("./lidbs"),
-  Dlda: require("./dlda")
+  Dlda: require("./dlda"),
+  ImportTnChecker: require("./importTnChecker")
 };
+
+for (const property in module.exports) {
```

Comment:
Clever girl

Reply:
![79e1c5b7b6cfd4ad9000b2213ab1daef](https://i.ytimg.com/vi/plzw1NWVYSM/maxresdefault.jpg)

Reply:
I like this approach, if there's a negative it's that someone adding additional functionality to the client wouldn't know to look for this abstraction and/or be aware that it's being applied, but I think it's straightforward enough to not be an issue.

Reply:
A little readme explanation should cover it while we wait on typescript